app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\User;
+use Illuminate\Support\Facades\Hash;
+use App\Http\Requests\FromSendRequest;
+
+class UserController extends Controller
+{
+    public function open_add_admin_page()
+    {
+        return view('admin.admin_register');
+    }
+    public function admin_register(FromSendRequest $request)
+    {
+        $admin = new User;
```

Comment:
You can make this short by using the ::create function in Laravel.

Please read about thesethis:
https://laravel.com/docs/5.7/eloquent#mass-assignment